presidio-analyzer/tests/test_analyzer_engine.py

```diff
@@ -9,15 +9,15 @@
 from analyzer.analyze_pb2 import AnalyzeRequest
 
 from analyzer import AnalyzerEngine, PatternRecognizer, Pattern, \
-    RecognizerResult, RecognizerRegistry
+    RecognizerResult, RecognizerRegistry, AnalysisExplanation
 from analyzer.predefined_recognizers import CreditCardRecognizer, \
     UsPhoneRecognizer, DomainRecognizer, UsItinRecognizer, \
     UsLicenseRecognizer, UsBankRecognizer, UsPassportRecognizer
 from analyzer.recognizer_registry.recognizers_store_api \
     import RecognizerStoreApi  # noqa: F401
 from analyzer.nlp_engine import SpacyNlpEngine, NlpArtifacts
 from analyzer.predefined_recognizers import IpRecognizer, UsSsnRecognizer
-from tests.mocks import MockNlpEngine
+from tests.mocks import MockNlpEngine, app_tracer_mock
```

Comment:
Why not import the class itself?
add AppTracerMock to the mock's \_\_init\_\_.py file

```suggestion
from tests.mocks import MockNlpEngine, AppTracerMock
```